3rdParty/velocypack/include/velocypack/Iterator.h

```diff
@@ -42,7 +42,7 @@ class ArrayIterator {
   ArrayIterator() = delete;
 
   explicit ArrayIterator(Slice slice)
-      : _slice(slice), _size(0), _position(0), _current(nullptr) {
+      : _slice(slice), _size(0), _position(0), _current(nullptr), _first(nullptr) {
```

Comment:
Is there a corresponding PR in the velocypack-repo?

Reply:
The code is already committed and merged in the velocypack repository.
arangodb is just lagging behind here :smile: